Server/BackEndMonitoringServer/include/CContainerOfProcesses.h

```diff
@@ -0,0 +1,32 @@
+#pragma once
+
+#include <unordered_map>
```

Comment:
Try to move all required include to cpp file

Reply:
Only necessary includes left, [commit](https://github.com/ITA-Dnipro/BackEndMonitoring/commit/28775d4e4e211a6985cecfb05174a510800a77d4#diff-0544ae2661f9747e70ea10c500758935c3f7d93fc11a7c69c55cadb18011b360).